tests/components/august/test_init.py

```diff
@@ -3,15 +3,57 @@
 from unittest.mock import MagicMock
 
 from homeassistant.components import august
+from homeassistant.exceptions import HomeAssistantError
 
 from tests.components.august.mocks import (
+    MockAugustApiFailing,
+    MockAugustComponentData,
     _mock_august_authentication,
     _mock_august_authenticator,
+    _mock_august_lock,
 )
 
 
+def test_get_lock_name():
+    """Get the lock name from August data."""
+    data = MockAugustComponentData(last_lock_status_update_timestamp=1)
+    lock = _mock_august_lock()
+    data.set_mocked_locks([lock])
+    assert data.get_lock_name("mockdeviceid1") == "Mocked Lock 1"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5020228)